src/vos/evtree.c

```diff
@@ -2883,7 +2882,7 @@ int evt_delete(daos_handle_t toh, const struct evt_rect *rect,
 		return rc;
 
 	if (ent_array.ea_ent_nr == 0)
-		return -DER_ENOENT;
+		return -DER_NONEXIST;
```

Comment:
Not a big deal, but why it's changed from ENOENT to NONEXIST?

Reply:
just because this is the only place in vos we return ENOENT

Reply:
we should remove one of those error code...